src/server.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+
+const logger = require('./middleware/logger.js');
+const errors = require('./error-handler/500.js');
+const notFound = require('./error-handler/404.js');
+
+//global -> app levle middleware
+//allways stick this in your server file
+app.use(express.json());
+app.use(looger); 
+
+//req.method = GET
+//req.path = '/person'
+//req.query = { name: "name provided"}
+
+
+//new way - with request parameter
+//http://localhost:3333/coach/football
+app.get('/person', (req, res) => {
+    res.send(`person ${req.query.user}`);
```

Comment:
change 'user' to 'name' because that what you are referring to in the validator middleware.